lingpy/tests/data/test_sound_class_models.py

```diff
@@ -7,7 +7,7 @@
 
 class Tests(object):
```

Comment:
Here, I suggest, removing it from tests, as model-tests are designed for the "basic" sound class models (we also do not include "jaeger" here, but have it as a model).